ios/FluentUI/Drawer/DrawerController.swift

```diff
@@ -123,13 +110,23 @@ open class DrawerController: UIViewController {
     }
 
     /// Set `backgroundColor` to customize background color of the drawer
-    @objc open var backgroundColor: UIColor = Colors.Drawer.background {
+    @objc open lazy var backgroundColor: UIColor = DrawerController.drawerBackgroundColor(fluentTheme: view.fluentTheme) {
         didSet {
             useCustomBackgroundColor = true
             view.backgroundColor = backgroundColor
         }
     }
 
+    @objc static func drawerBackgroundColor(fluentTheme: FluentTheme) -> UIColor {
```

Comment:
Can these two static functions be `private`? Doesn't look like they're referenced anywhere else.